content/300-guides/050-database/100-developing-with-prisma-migrate/150-team-development.mdx

```diff
@@ -102,8 +102,8 @@ Your team members Ania and Javier make additive changes to the schema in their l
 
 1. Commits the changed schema and the new migration:
 
-      * `./prisma/schema.prisma`
-      * `./prisma/migrations/20210316081837-new-field/migration.sql`
+   - `./prisma/schema.prisma`
+   - `./prisma/migrations/20210316081837-new-field/migration.sql`
```

Comment:
Does this render correctly?

Reply:
Looks good

<img width="702" alt="image" src="https://user-images.githubusercontent.com/4734297/124765651-9cbd6380-df4f-11eb-92ae-6cb187b9ea3b.png">